docs/what-is-ethereum/index.md

```diff
@@ -8,9 +8,53 @@ meta:
   - property: og:description
     content: Guides, resources, and tools for users completely new to Ethereum.
 lang: en-US
----
+layout: landingpage
```

Comment:
In general I'd prefer to keep this type of content within an e.g. `<WhatIsEthereumPage/>` component with the strings.json translation pattern vs. having all the content in the markdown. The primary reason is that it will lead to less duplication, e.g. imagine this is rolled out on 30 languages. If we want to change the image for one of these cards, now we have to update the card's `header` in 30 places vs. in 1 component file. Or if we decide to remove the card `title` for whatever reason or make the card clickable, it'd be best to consolidate that in one place to edit vs. across a bunch of markdown files.